crates/ide_db/src/helpers.rs

```diff
@@ -282,3 +282,21 @@ pub fn for_each_break_expr(
         }
     }
 }
+
+/// Checks if the given lint is affected by the other lint which may or may not be a group.
+pub fn lint_is_affected(lint: &str, lint_is: &str) -> bool {
```

Comment:
not a fan of this naming. bikeshed appreciated

Reply:
`lint_eq_or_in_group` maybe?